src/main/java/shipit/later/BucketDownloader.java

```diff
@@ -80,18 +81,15 @@ public void download(String outputPath) throws IOException {
 			// 4: Iterate over the Data and print out the full document
 			for (ViewRow row : result) {
 				try {
-					String csvLine = row.id() + ","
-							+ row.document().content().toString();
+					String csvLine = row.id() + "," + row.document().content().toString();
 					bos.write(csvLine.getBytes());
 					bos.write("\n".getBytes());
+				} catch (TranscodingException e) {
+					kvBos.write((row.key().toString() + "," + row.document(RawJsonDocument.class).content()).getBytes());
+					kvBos.write("\n".getBytes());
 				} catch (Exception e) {
-					// System.out.println("Error: " + e);
-					if (e instanceof com.couchbase.client.java.error.TranscodingException) {
-						kvBos.write((row.key().toString() + ","
-								+ row.document(StringDocument.class).content())
-										.getBytes());
-						kvBos.write("\n".getBytes());
-					}
+					// do nothing
+				    System.out.println("Error: " + e);
 				}
```

Comment:
So it was `row.document(StringDocument.class)` throwing exception. I switched to `RawJsonDocument` because same used in https://github.com/vchau/couch-downloader/blob/master/src/main/java/shipit/later/NonJsonDocumentUploader.java#L33